### PR TITLE
Project started with folders and completed Navbar

### PR DESCRIPTION
**Pull request by MarioPaju1991**
Completed Navbar with CSS and it responsive to different screen sizes

### CODE REVIEW

**Review by nagamine-git — COMMENTED** [review]
I strongly recommend that you include the following
- What did you do? Screenshots help reviewers understand what you did.
- What did you test?
    - `- [ ]` Checkbox helps your testing.
- Use a unique branch name.
    - E.g. `add_navbar`, `feature/add_navbar`, or  use [this](https://github.blog/changelog/2022-03-02-create-a-branch-for-an-issue/)
    - This time is OK.

**Comment by MarioPaju1991** [comment]
image

Navbar created for Homepage

Tests
- [ OK ] Navbar created
- [ OK ] Navbar links visible / clickable
- [ OK ] Navbar buttons visible 
- [ OK ] Searchbar created
- [ OK ] Logo visible

**Comment by nagamine-git** [comment]
> image
> 
> Navbar created for Homepage
> 
> Tests
> 
> * [ OK ] Navbar created
> * [ OK ] Navbar links visible / clickable
> * [ OK ] Navbar buttons visible
> * [ OK ] Searchbar created
> * [ OK ] Logo visible

Great! Also, you can use them.
- [x] 
- [ ]

**Review by nagamine-git — APPROVED** [review]
LGTM

File: src/app/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <ul className={styles.navLinks}>
+        {navLinks.map(link =>
+          <li key={link.href}>
+            <Link href={link.href}>{link.label}</Link>
+          </li>
+        )}
+      </ul>
+      <div className={styles.buttons}>
+        <button className={styles.login}>Login</button>
+        <button className={styles.login}>Sign Up</button>
```
Comment (kuwahalab) [inline src/app/components/Navbar.js:31]: If it is not set intentionally, it is better to set different values ​​for the class name, in this case login and signup.

**Review by kuwahalab — COMMENTED** [review]
I'm sorry for being late. I left one comment. But there were almost no problems!

File: src/app/components/Navbar.js (hunk 1 above)
Comment (MarioPaju1991) [reply src/app/components/Navbar.js:31]: True ill change it and keep it separetly

**Pull request merged**